scheduler/docs/configuration.asc

```diff
@@ -98,6 +98,39 @@ We'll look at the configurable options in turn:
   This sets the role that Cook will connect to Mesos with. Default: `*`
   You can omit this property unless you've enabled security features in Mesos. The value should be in authorized list for the current `:principal` in `register_frameworks` Action in Mesos Authorization file. See http://mesos.apache.org/documentation/latest/authorization/ for details.
 
+`:container`::
```

Comment:
Can you move this to the [scheduler api doc](https://github.com/mojotech/Cook/blob/d356a82bde4b8dc18876db600627b13d0ecf123a/scheduler/docs/scheduler-rest-api.asc#launch-a-job) and follow the same format?